tools/ui-components/src/form-control/form-control.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { FormControlFeedback as Feedback } from './form-control-feedback';
+import { FormControlStatic as Static } from './form-control-static';
+import { FormControlProps } from './types';
+
+const FormControl = React.forwardRef<HTMLInputElement, FormControlProps>(
+  ({ className, id, testId }, ref): JSX.Element => {
+    const defaultClasses =
+      'outline-0 block w-full h-8 py-1.5 px-2.5 text-md text-default-foreground-primary ' +
+      'bg-default-background-primary bg-none rounded-none border-1 border-solid ' +
+      'border-default-background-quaternary shadow-none ' +
+      'transition ease-in-out duration-150 focus:border-default-foreground-tertiary';
```

Comment:
These are derived from the used bootstrap and learn custom styles that target the .form-control class and its focus variation.